### ISSUE TITLE
Changelog? 

### ISSUE THREAD

**Issue by chromaniac2** [PageMonitorPlus/pagemon-chrome-ext#1]
Just noticed that it was updated on the Chrome Web Store. A Changelog would be nice. Thanks.

**Comment by PageMonitorPlus** [comment]
Page Monitor Plus was taken down after Serge from Visualping sent a misleading "abuse" report. The last update was to restore it back after I emailed Google back. Nevertheless it is a good idea to add a changelog. Will add one when a change is made.

**Comment by chromaniac2** [comment]
Thanks!

**Comment by AndreeTor** [comment]
Can you upload extension to Opera addons page?

**Comment by chromaniac2** [comment]
So... It has been removed from Chrome App Store again? How do I install it?

**Comment by madourrhani** [comment]
Hey Sushubh, they put the extension back on https://chrome.google.com/webstore/detail/page-monitor/ogeebjpdeabhncjpfhgdibjajcajepgg. It's 332 kb and doesn't seem to contain the Amplitude Tracker.

I talked to Serge (a nice guy that has a Harvard degree but doesn't know much about coding - as everybody noticed) and they are not ill intended... just plain rockies. More importantly, and after having been burned so badly by the botched migration, he promised not to touch the version they posted there.

The app can also be downloaded at http://www.crx4chrome.com/crx/1257/ but this version is 837 kb so pressumably it contains Amplitude. I checked for older versions but can't find any. Hope this helps, Madour
